src/test/java/integ/global_attribute/GlobalAttributeProcess.java

```diff
@@ -0,0 +1,213 @@
+                .addColumn(PersistenceTableColumnSchema.create(COL_KEY_1, String.class, true))
+                .addColumn(PersistenceTableColumnSchema.create(COL_KEY_2, String.class, true))
+                .addColumn(PersistenceTableColumnSchema.create(COL_KEY_3, Integer.class, false))
+        );
+    }
+
+    @Override
+    public ProcessOptions getOptions() {
+        return ProcessOptions
+            .builder(GlobalAttributeProcess.class)
+            .processStartConfig(
+                ProcessStartConfig
+                    .builder()
+                    .build()
+                    .addGlobalAttributesToUpsert(
+                        PersistenceTableRowToUpsert
+                            .create(TABLE_NAME, AttributeWriteConflictMode.RETURN_ERROR_ON_CONFLICT)
+                            .addPrimaryKeyColumn(PK_KEY, PK_VALUE)
+                            .addNonPrimaryKeyColumn(COL_KEY_1, COL_VALUE_1)
+                            .addNonPrimaryKeyColumn(COL_KEY_2, COL_VALUE_2)
+                    )
+            )
+            .build();
+    }
+}
+
+class GlobalAttributeProcessStartingState implements AsyncState<Void> {
+
+    @Override
+    public Class<Void> getInputType() {
+        return Void.class;
+    }
+
+    @Override
+    public CommandRequest waitUntil(final Context context, final Void input, final Communication communication) {
+        System.out.println("GlobalAttributeProcessStartingState.waitUntil: " + input);
+
+        return CommandRequest.EMPTY;
+    }
+
+    @Override
+    public StateDecision execute(
+        final Context context,
+        final Void input,
+        final CommandResults commandResults,
+        final Persistence persistence,
+        final Communication communication
+    ) {
+        System.out.println("GlobalAttributeProcessStartingState.execute: " + input);
+
+        assertEquals(PK_VALUE, persistence.getGlobalAttribute(TABLE_NAME, PK_KEY));
+        assertEquals(COL_VALUE_1, persistence.getGlobalAttribute(TABLE_NAME, COL_KEY_1));
+        assertEquals(COL_VALUE_2, persistence.getGlobalAttribute(TABLE_NAME, COL_KEY_2));
+        assertThrows(
+            GlobalAttributeNotFoundException.class,
+            () -> persistence.getGlobalAttribute(TABLE_NAME, COL_KEY_3)
+        );
+
+        persistence.upsertGlobalAttribute(TABLE_NAME, COL_KEY_1, COL_VALUE_1_2);
+        persistence.upsertGlobalAttribute(TABLE_NAME, COL_KEY_3, COL_VALUE_3);
+
+        return StateDecision.singleNextState(GlobalAttributeProcessNextState1.class, null);
+    }
+}
+
+class GlobalAttributeProcessNextState1 implements AsyncState<Void> {
+
+    @Override
+    public Class<Void> getInputType() {
+        return Void.class;
+    }
+
+    @Override
+    public AsyncStateOptions getOptions() {
+        return AsyncStateOptions
+            .builder(GlobalAttributeProcessNextState1.class)
+            .persistenceSchemaToLoad(
```

Comment:
in iWF, we asked user to define the loading schema and which makes it hard to use, especially for RPC. 

I am thinking to replace it with the named persistence policy approach. User will define a set of persistence policy with a name and use them in stateOptions and RPC annotation parameter directly. 

Also, it's renamed from persistence loading policy to persistence policy, because it is not only shorter, but will probably include something  related to write (like how to handle write conflict) in the future.

Reply:
What is  the named persistence policy approach?

Reply:
> What is the named persistence policy approach?

https://github.com/xdblab/xdb-java-sdk/pull/12#discussion_r1393419406